src/devices/machine/scnxx562.cpp

```diff
@@ -888,6 +890,10 @@ uint8_t duscc_channel::do_dusccreg_ctcr_r()
 void duscc_channel::do_dusccreg_ctcr_w(uint8_t data)
 {
 	LOG("%s(%02x) -  not supported yet\n", FUNCNAME, data);
+	if ((m_ctcr ^ data) == REG_CTCR_ZERO_DET_INT && !(data & REG_CTCR_ZERO_DET_INT))
+	{
+		m_uart->clear_interrupt(m_index, INT_EXTCTSTAT);
+	}
```

Comment:
This can’t be right – you’re testing that the *only* change to the register contents is the zero detect interrupt flag (bit 7).  I can’t find anything in the manual to suggest it ignores a change to this bit if other bits change at the same time.